projects/plugins/jetpack/extensions/shared/components/product-management-controls/store/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import * as actions from './actions';
+import reducer from './reducer';
+import * as resolvers from './resolvers';
+import * as selectors from './selectors';
+
+export const jetpackMembershipProductsStore = 'jetpack/membership-products';
+
+const store = registerStore( jetpackMembershipProductsStore, {
```

Comment:
Register store is deprecated, I think we need to use register instead. [Github link](https://github.com/WordPress/gutenberg/blob/9d0689b7bf83688d02ede386e3d2e7d308de8d29/packages/data/src/index.js#L180)

Reply:
I've updated this, and actually took the chance of moving the store to a shared, centralized location, along with `jetpack/media-source`.

Now the store is actually only registered once (it wasn't a big deal, but still), and the overall implementation is consistent with Core Gutenberg (using the actual store as `StoreDescriptor` for selectors and dispatchers).